src/pybind/mgr/prometheus/module.py

```diff
@@ -1138,6 +1155,14 @@ def get_service_list(self) -> Dict[Tuple[str, str], Tuple[str, str, str]]:
     @profile_method()
     def get_metadata_and_osd_status(self) -> None:
         osd_map = self.get('osd_map')
+
+        cluster_nearfull_ratio = osd_map.get('nearfull_ratio', None)
+        cluster_full_ratio = osd_map.get('full_ratio', None)
```

Comment:
This is still using osd map so I want to understand how this will not affect from prometheus ?
Thanks

Reply:
I don't have answer to this: @adk3798 , any thoughts?

I assume the frequency is lower(5 vs 15 sec)